cmd/cainjector/app/start.go

```diff
@@ -55,23 +56,23 @@ func (o *InjectorControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		"If set, this limits the scope of cainjector to a single namespace. "+
 		"If set, cainjector will not update resources with certificates outside of the "+
 		"configured namespace.")
-	fs.BoolVar(&o.LeaderElect, "leader-elect", true, ""+
+	fs.BoolVar(&o.LeaderElect, "leader-elect", cmdutil.DefaultLeaderElect, ""+
 		"If true, cainjector will perform leader election between instances to ensure no more "+
 		"than one instance of cainjector operates at a time")
-	fs.StringVar(&o.LeaderElectionNamespace, "leader-election-namespace", "", ""+
+	fs.StringVar(&o.LeaderElectionNamespace, "leader-election-namespace", cmdutil.DefaultLeaderElectionNamespace, ""+
 		"Namespace used to perform leader election (defaults to controller's namespace). "+
 		"Only used if leader election is enabled")
```

Comment:
This documentation needs updating. The new default should  appear  in the `--help` output.

Reply:
Good catch.

It’s updated:

```
      --leader-election-namespace string          Namespace used to perform leader election. Only used if leader election is enabled (default "kube-system")
```